src/main/java/com/epam/bookstoreservice/mapper/UserResponseDTOAssembler.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.bookstoreservice.mapper;
+
+import com.epam.bookstoreservice.dto.response.UserResponseDTO;
+import com.epam.bookstoreservice.model.UserModel;
+import org.springframework.hateoas.server.RepresentationModelAssembler;
+import org.springframework.stereotype.Component;
+
+/**
+ * model assembler for userResponseDTO
+ */
+@Component
+public class UserResponseDTOAssembler implements RepresentationModelAssembler<UserResponseDTO,UserModel> {
```

Comment:
You could move this assembler to a new package called 'hateoas/assembler' since this feature is related to it.